rules/web/web_exchange_proxyshell.yml

```diff
@@ -32,7 +33,7 @@ detection:
             - '%3f@foo.com'
             - 'Email=autodiscover/autodiscover.json'
             - 'json?@foo.com'
-    condition: selection_auto and selection_uri or selection_poc 
+    condition: selection and ( selection_auto and selection_uri or and selection_poc )
```

Comment:
or and selection_poc ?

To keep condition simple , add the `sc-status` in the two part

Reply:
I changed it